<ui version="4.0" >
 <class>settings</class>
 <widget class="QWidget" name="settings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>155</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Pyramid</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="4" column="1" >
       <widget class="QDoubleSpinBox" name="heightPyramid" >
        <property name="decimals" >
         <number>4</number>
        </property>
        <property name="maximum" >
         <double>1000.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>0.010000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QDoubleSpinBox" name="radiusPyramid" >
        <property name="decimals" >
         <number>4</number>
        </property>
        <property name="maximum" >
         <double>1000.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>0.010000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="cornersPyramid" >
        <property name="maximum" >
         <number>1000</number>
        </property>
        <property name="minimum" >
         <number>3</number>
        </property>
        <property name="value" >
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2" >
       <widget class="QCheckBox" name="centeredPyramid" >
        <property name="text" >
         <string>Centered</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2" >
       <widget class="QCheckBox" name="sharedVerticesPyramid" >
        <property name="text" >
         <string>Shared Vertices</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="maximumSize" >
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="maximumSize" >
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>Radius:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="maximumSize" >
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>Corners:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2" >
       <widget class="QPushButton" name="insertButton" >
        <property name="text" >
         <string>Insert Pyramid</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cornersPyramid</tabstop>
  <tabstop>radiusPyramid</tabstop>
  <tabstop>heightPyramid</tabstop>
  <tabstop>sharedVerticesPyramid</tabstop>
  <tabstop>centeredPyramid</tabstop>
  <tabstop>insertButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
